site/src/templates/standard-article.js

```diff
@@ -161,7 +163,19 @@ export default class Article extends React.Component {
     }
     return false
   }
-
+  hasCCNote(post) {
```

Comment:
misleading method name. It's actually "shouldRenderCCNoteAndHasCCNote" and that implies an incorrect separation of concerns.  Split into 2 methods to clarify.

Reply:
done